appservice/src/deploy/deployJar.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Please remove these files/changes related to this PR: https://github.com/microsoft/vscode-azuretools/pull/912.  I don't believe that this PR is dependent on these changes so I'd rather keep them separate for ease of reviewing.

Reply:
the corresponding commit has been dropped from this branch.